microsoftgraph/microsoft-graph-explorer-v4
# Fix: Adaptive cards and toolkit heights

## Overview
#2703 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Click on the toolkit tab. Notice that the height is okay
Click on the Run query button to run /me
Click on the Adaptive cards tab and check that JSON tab height is okay
